pkg/solana/fees/computebudget.go

```diff
@@ -151,6 +151,17 @@ func set(tx *solana.Transaction, baseData instruction, appendToFront bool) error
 
 		// https://github.com/gagliardetto/solana-go/blob/618f56666078f8131a384ab27afd918d248c08b7/transaction.go#L293
 		tx.Message.Header.NumReadonlyUnsignedAccounts++
+
+		// lookup table addresses are indexed after the tx.Message.AccountKeys
+		// higher indices must be increased
+		// https://github.com/gagliardetto/solana-go/blob/da2193071f56059aa35010a239cece016c4e827f/transaction.go#L440
```

Comment:
If solana-go lib does the same, why do we need a wrapper here? Can not we use just the library instead? I'm thinking that we need to avoid having any other bug on our side in the future

Reply:
it doesn't do the exact same thing - this comment is mostly just pointing out that lookup table addresses are indexes that are larger than the normal message.accountkeys array

solana-go requires a user to predeclare all instructions and then compile things together
our utility allows modification of an compiled transaction to include compute budget instructions